changelogs/CHANGELOG.rst

```diff
@@ -5,6 +5,24 @@ Cisco Iosxr Collection Release Notes
 .. contents:: Topics
 
 
+v1.1.0
+======
+
+Major Changes
```

Comment:
It'll be a minor change.

Reply:
As discussed, major changes involve backward compatible ones and need a major version bump.